### ISSUE TITLE
VideoContext responsiveness 🐛

### ISSUE THREAD

**Issue by pietrop** [bbc/digital-paper-edit-client#78]
**Describe the bug**

VideoContext preview of paper edit is not responsive

**To Reproduce**
1. Go to online demo [projects-->paperedits](https://bbc.github.io/digital-paper-edit-client/#/projects/10046281c4ad4938b7d0ae6fa9899bec/paperedits/86e938c468de4b0cb873c6e97a3a25f8) (on chrome)
2. See error

**Expected behavior**

This is what it looked like before - screenshot from electron app `Version 1.0.9-alpha.1 `
<img width="1112" alt="Screenshot 2019-10-18 at 12 39 02" src="https://user-images.githubusercontent.com/4661975/67091337-4836eb00-f1a4-11e9-8979-1bf693587a57.png">

**Screenshots**

<img width="1449" alt="Screenshot 2019-10-18 at 12 36 13" src="https://user-images.githubusercontent.com/4661975/67091272-1f165a80-f1a4-11e9-91aa-06260f5f046b.png">

**Additional context**

[previewcanvas--videocontext](https://bbc.github.io/digital-paper-edit-storybook/?path=/story/previewcanvas--videocontext) in [bbc.github.io/digital-paper-edit-storybook](https://bbc.github.io/digital-paper-edit-storybook/)

Also there is CORS error in the demo (probl separate issue?)

```
Access to video at 'https://digital-paper-edit-demo.s3.eu-west-2.amazonaws.com/PBS-Frontline/The+Facebook+Dilemma+-+interviews/The+Facebook+Dilemma+-+Soleio+Cuervo-OIAUfZBd_7w.mp4' from origin 'https://bbc.github.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
I think that might be the CORS settings in S3

**Comment by emettely** [comment]
Thanks for raising the bug (no sarcasm :D)
I'm on firefox and I don't see that issue. I'll double check on other browsers - assuming you are on Chrome.

**Comment by emettely** [comment]
Ok, so I can see it in Chrome. Will have a proper gander and get back to you.

**Comment by emettely** [comment]
So it is fixed, but I seem to have maybe introduced a new problem with some dependency updates, and the demo page is now down (see #79) ... :(
I am going to close this ticket, since the fix is demonstrated + there is another ticket with the addressed bug above. Will create a separate ticket Re the CORS issue (having looked at S3, the CORS is present - so more to look into).

# Diff
https://github.com/bbc/digital-paper-edit-client/compare/46e877c7f30a9f546475d00e039a6706a2e6bfa7...master

# Screenshots

<img width="791" alt="Screen Shot 2019-10-19 at 12 18 15" src="https://user-images.githubusercontent.com/8188418/67144085-af1ed780-f26a-11e9-8dd9-e80fc442768d.png">
<img width="573" alt="Screen Shot 2019-10-19 at 12 18 25" src="https://user-images.githubusercontent.com/8188418/67144086-af1ed780-f26a-11e9-916e-0dfb2939e8d0.png">
